testing/mutter/APKBUILD

```diff
@@ -2,10 +2,10 @@
 # Maintainer: Rasmus Thomsen <oss@cogitri.dev>
 pkgname=mutter
 pkgver=3.32.2
-pkgrel=0
+pkgrel=1
 pkgdesc="clutter-based window manager and compositor"
 url="https://wiki.gnome.org/Projects/Mutter/"
-arch="all !s390x !ppc64le" # limited by gnome-settings-daemon
+arch="all !s390x !armv7" # limited by gnome-settings-daemon
```

Comment:
Why have you added armv7 to the list here?

Reply:
drone exposed that armv7 build of mutter will fail because of a failing dependency on the libgweather apk package. Looking at the edge repository community/libgweather isn't available for armv7.

Reply:
That's due to a temporary failure in the armv7 builder, which should he fixed "soon"™. Please leave armv7 enabled and ignore the CI failure.

Reply:
Thanks Cogitri. Sounds good. I'll update the PR and re-enable armv7.

Reply:
Thank you :)